documentation/user/how-to-report-an-issue.md

```diff
@@ -11,6 +11,8 @@ parent: wiki
 
 If you've found an issue with ACE3 please make sure that ACE3 is really the cause of the problem. To do this try to reproduce the issue with using only `@cba_a3` and `@ACE3` on a newly created mission.
 
+Indicate if the issue appears on stable or dev, if dev indicate the commit the version is based on.
```

Comment:
Well every file has a commit hash 
 so just saying that that would be enough. Bikey for example.